### ISSUE TITLE
User roles disappear

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#507]
```
What steps will reproduce the problem?
1.Assign a role to a user
2.Save the user
3.Click "Cases"

What is the expected output? 
A case can be launched that later will be assigned to the role.
What do you see instead?
The roles that were assigned to the user have dissapeared in the "role" "owns:" 
field for the user


What version of the product are you using? On what operating system?

YAWL4Study - 2.3.5final
Please provide any additional information below.

N.B. even when the roles have disappeared as discribed above, when the roles 
are inspected in the organizational data enverionment, YAWL will state that the 
roles is still assigned.

I added the export file of the organizational data right after the problem 
occured

```

Original issue reported on code.google.com by `Mark.J.S...@gmail.com` on 20 Mar 2014 at 1:48

Attachments:
* [YAWLOrgDataExport (1).ybkp](https://storage.googleapis.com/google-code-attachments/yawl/issue-507/comment-0/YAWLOrgDataExport (1).ybkp)

**Comment by yawlfoundation** [comment]
```
Also, the logs don't show any errors
```

Original comment by `Mark.J.S...@gmail.com` on 20 Mar 2014 at 1:51

**Comment by yawlfoundation** [comment]
```
fixed in 3.0beta - to be released in next couple of days, or can update 
existing Y4S from here: http://www.yawlfoundation.org/pages/resources/yawl3.html

```

Original comment by `yawl.mic...@gmail.com` on 21 Mar 2014 at 6:58
* Changed state: **Fixed**
